lib/index.d.ts

```diff
@@ -0,0 +1,17 @@
+interface Options {
+  timeout?: number;
+  verifyDomain?: boolean;
+  verifyMailbox?: boolean;
+}
+
+interface VerifyResult {
+  wellFormed: boolean;
+  validDomain: boolean;
```

Comment:
```suggestion
  validDomain: boolean | null;
```

Will be `null` if `verifyDomain` is `false`